### PR TITLE
Add: Adding initial documentation for keyboard events (JavaScript)

### PR DESCRIPTION
**Pull request by harsha509**

### CODE REVIEW

**Comment by harsha509** [comment]
hi @diemol, Can you please review this PR

**Comment by diemol** [comment]
Could you please add the text to the other languages?
I am also having trouble with the phrase:
`KeyBoard actions are performed by using low-level
interface which allows us to provide virtualized device input to the web browser`
where does it come from?

**Comment by harsha509** [comment]
@diemol , Am unsure to get started with a phrase as keyboard.md was empty, so made of my own. I need your inputs to change the phrase too. Once the phrase gets finalized i can update the translated code to other languages.

let me know your thoughts on this.

File: docs_source_files/content/webdriver/keyboard.fr.md [hunk 1]
```diff
@@ -7,4 +7,46 @@ weight: 10
...
+  {{< code-panel language="javascript" >}}
+const {Builder, By, Key} = require('selenium-webdriver');
+
+(async function example() {
+
+let driver = await new Builder().forBrowser('chrome').build();
+
+try {
+
+// Navigate to Url
```
Comment (diemol) [inline docs_source_files/content/webdriver/keyboard.fr.md]: Seems there is no indentation, could you please indent the code blocks?

**Review by diemol — CHANGES_REQUESTED** [review]

File: docs_source_files/content/webdriver/keyboard.fr.md (hunk 1 above)
Comment (harsha509) [reply docs_source_files/content/webdriver/keyboard.fr.md]: hi @diemol , updated indentation for all code samples

Comment (diemol) [reply docs_source_files/content/webdriver/keyboard.fr.md]: 👍

File: docs_source_files/content/webdriver/keyboard.en.md [hunk 2]
```diff
@@ -3,3 +3,42 @@ title: "Keyboard"
 weight: 10
 ---
 
+Represents a KeyBoard event. KeyBoard actions are performed by using low-level
+interface which allows us to provide virtualized device input to the web browser.
+
+# sendKeys
```
Comment (diemol) [inline docs_source_files/content/webdriver/keyboard.en.md:9]: Something I missed, and it is already merged, this should not be an `# sendKeys` but instead a `## sendKeys`.

Could you please send another PR with that change @Harsha509? Thanks!

Comment (harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md:9]: Sure. i'll send another PR.

Comment (harsha509) [reply docs_source_files/content/webdriver/keyboard.en.md:9]: @diemol, Updated in PR https://github.com/SeleniumHQ/docs/pull/351

I'll be cautious next time.

Thanks!
